service/src/main/resources/HTMLTemplate.html

```diff
@@ -1249,7 +1254,7 @@ <h3 class="heading heading--two">Respondent {{counter}}</h3>
           </dd>
         </div>
         <div class="data-list__row">
-          <dt class="data-list__title">Address</dt>
+          <dt class="data-list__title">Current Address</dt>
```

Comment:
Shouldn't address be lowercase? I am not a content guy so just asking :)

Reply:
maybe so ^ see comment above, I standardised it to C and A. Happy to change to C and a instead. Needs to be one of the other

Reply:
Raise with Mia

Reply:
Checked with Mia, it should be Ca. Please change.